bindings/wasm/examples/browser/main.js

```diff
@@ -44,3 +45,8 @@ document
 document
     .querySelector("#merkle_key_btn")
     .addEventListener("click", () => merkleKey(clientConfig));
+
+//handle private tangle DID creation on click event
+document
+    .querySelector("#private_tangle_btn")
+    .addEventListener("click", () => createIdentityPrivateTangle());
```

Comment:
What do we actually expect to happen when a user runs a private tangle example? There will always be an error in the browser console but it's unclear and not printed to the text box on the screen.

Maybe the browser example can have a text input for the private network name and node address similar to the resolve DID example?